src/Cli/dotnet/Commands/Build/BuildCommand.cs

```diff
@@ -61,4 +86,21 @@ public static int Run(ParseResult parseResult)
 
         return FromParseResult(parseResult).Execute();
     }
+
+    public abstract int Execute();
+}
+
+public sealed class ForwardingBuildCommand(
```

Comment:
Nit: naming. I now get the intent of `ForwardingBuildCommand` as in it 'forwards the arguments on to something else' but found it confusing at first. Consider calling it `ExternalBuildCommand` or something to more clearly delineate the difference between it and the virtual one.

Reply:
Removed this class altogether by addressing your other comment.